kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8/kafkaproxies.kroxylicious.io-v1.yml

```diff
@@ -51,16 +51,41 @@ spec:
             spec:
               type: object
               properties:
-                podTemplate:
+                infrastructure:
                   type: object
                   properties:
-                    metadata:
+                    labels:
```

Comment:
I wonder if we should defer implementing `labels`.   

I think SSA would be an option for a user who want to **annotate** (not label) their LB service with Cloud Provider specific settings ([example](url)).

I think we wait to hear some concrete requirements.  Do people want to annotate/label everything? Or just a subset.